src/App.tsx

```diff
@@ -15,6 +15,7 @@ import WalletProvider from './components/WalletProvider';
 import AppRouterLayout from './layout/AppRouterLayout';
 import Loading from './pages/Loading';
 import NotFound from './pages/NotFound';
+import {MathJaxContext} from 'better-react-mathjax';
```

Comment:
Strange, but ok. Guess for memoizing or something.

Reply:
Think the auto-format didn't trigger, re-formatted the file